CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ Version number changes (major.minor.micro) in this package denote the following:
 
 ## Unreleased
 
+## [3.0.1] - XXXX-XX-XX
```

Comment:
If it's not for immediate release, then this PR shouldn't include any version numbers. But we might as well make a release. Please put today's date here.